tee-worker/omni-executor/rpc-server/src/detailed_error.rs

```diff
@@ -0,0 +1,197 @@
+use jsonrpsee::types::ErrorObject;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct DetailedError {
+	pub code: i32,
+	pub message: String,
+	pub details: ErrorDetails,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ErrorDetails {
+	#[serde(skip_serializing_if = "Option::is_none")]
+	pub field: Option<String>,
```

Comment:
What's the meaning of the `field` here and how it should be used ? 
Why in some cases it's populated with `operation` ?

Reply:
The `field` is meant for invalid field params. I'll remove the usage with operation